Lesson9/src/TextFormater.java

```diff
@@ -0,0 +1,72 @@
+import java.io.*;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class TextFormater implements Reader {
+    String str;
+    String[] words;
+    int count1 = 0;
+    int count2 = 0;
+    int count3 = 0;
+
+    @Override
+    public void readFile() {
```

Comment:
please refactor this method to have a String param with fileName, don't read filename from console inside